CHANGELOG.md

```diff
@@ -2,14 +2,21 @@
 
 ## v0.6.0 (TBD)
 
+- Added `AccountCode` as part of `GetAccountProofs` endpoint response (#521).
```

Comment:
nit: we should move this to the end of the list as we usually list changes with earliest on top.

Reply:
Hmm. This is a bit concerning tbh - I would have thought post-merge the result should match `next` exactly. However even with your commit there are still changes between this branch and `next's changelog. The ordering is different:

`next`:
```
## v0.6.0 (TBD)

- Added `AccountCode` as part of `GetAccountProofs` endpoint response (#521).
- [BREAKING] Added `kernel_root` to block header's protobuf message definitions (#496).
- [BREAKING] Renamed `off-chain` and `on-chain` to `private` and `public` respectively for the account storage modes (#489).
- Optimized state synchronizations by removing unnecessary fetching and parsing of note details (#462).
- [BREAKING] Changed `GetAccountDetailsResponse` field to `details` (#481).
- Improve `--version` by adding build metadata (#495).
- [BREAKING] Introduced additional limits for note/account number (#503).
- [BREAKING] Removed support for basic wallets in genesis creation (#510).
- Added `GetAccountProofs` endpoint (#506).
- Migrated faucet from actix-web to axum (#511).
- Changed the `BlockWitness` to pass the inputs to the VM using only advice provider (#516).
- [BREAKING] Improved store API errors (return "not found" instead of "internal error" status if requested account(s) not found) (#518).
- [BREAKING] Migrated to v0.11 version of Miden VM (#528).
- Reduce cloning in the store's `apply_block` (#532).
- [BREAKING] Changed faucet storage type in the genesis to public. Using faucet from the genesis for faucet web app. Added support for faucet restarting without blockchain restarting (#517).
```

this branch:
```
## v0.6.0 (TBD)

- Optimized state synchronizations by removing unnecessary fetching and parsing of note details (#462).
- [BREAKING] Changed `GetAccountDetailsResponse` field to `details` (#481).
- Improve `--version` by adding build metadata (#495).
- [BREAKING] Added `kernel_root` to block header's protobuf message definitions (#496).
- [BREAKING] Renamed `off-chain` and `on-chain` to `private` and `public` respectively for the account storage modes (#489).
- [BREAKING] Introduced additional limits for note/account number (#503).
- [BREAKING] Removed support for basic wallets in genesis creation (#510).
- Added `GetAccountProofs` endpoint (#506).
- Migrated faucet from actix-web to axum (#511).
- Changed the `BlockWitness` to pass the inputs to the VM using only advice provider (#516).
- [BREAKING] Changed faucet storage type in the genesis to public. Using faucet from the genesis for faucet web app. Added support for faucet restarting without blockchain restarting (#517).
- [BREAKING] Improved store API errors (return "not found" instead of "internal error" status if requested account(s) not found) (#518).
- Added `AccountCode` as part of `GetAccountProofs` endpoint response (#521).
- [BREAKING] Migrated to v0.11 version of Miden VM (#528).
- Reduce cloning in the store's `apply_block` (#532).
```

Reply:
I think I'll try a rebase instead.

Reply:
I think it was my commit that introduced these changes. Before it, they should have been the same?

Reply:
Ah. Right; I confused myself with the github preview of your comment it seems.